lib/components/helpers/pretty-text-input.js

```diff
@@ -40,7 +40,11 @@ export default React.createClass({
   },
 
   componentDidUpdate: function(prevProps, prevState) {
-    if (prevState.codeMirrorMode !== this.state.codeMirrorMode) {
+    const replaceChoicesHaveChanged = prevProps.replaceChoices !== this.props.replaceChoices;
```

Comment:
Really minor thing, but I making the boolean word a prefix, so maybe `hasChangedReplaceChoices` instead?